astropy/io/votable/converters.py

```diff
@@ -160,7 +160,7 @@ class Converter:
     """
 
     def __init__(self, field, config=None, pos=None):
-        pass
+        self._config = config
```

Comment:
This should fix what I think is a real bug that didn't manifest itself before due to the different default of emitting warnings. This behavior makes sense anyway - if a config is passed to the initializer, it should be the default for the method calls after unless overridden.

Reply:
Actually I reverted this since it's not actually needed anymore